tests/TestCase/Controller/Traits/RegisterTraitTest.php

```diff
@@ -123,17 +123,22 @@ public function testRegisterWithEventFalseResult()
      */
     public function testRegisterWithEventSuccessResult()
     {
-        $this->assertEquals(0, $this->table->find()->where(['username' => 'testRegistration'])->count());
-        $this->_mockRequestPost();
-        $this->_mockAuth();
-        $this->_mockFlash();
-        $this->_mockDispatchEvent(new Event('Users.Component.UsersAuth.beforeRegister'), [
+        $data = [
             'username' => 'testRegistration',
             'password' => 'password',
```

Comment:
does the test fail before the change?

Reply:
the test started failing after the change